charts/redhat/redhat/redhat-dotnet-imagestreams/0.0.2/src/Chart.yaml

```diff
@@ -0,0 +1,16 @@
+description: |-
+  Build and run .NET applications on UBI.
+  For more information about using this builder image, including OpenShift considerations,
+  see https://github.com/redhat-developer/s2i-dotnetcore.
+annotations:
+  charts.openshift.io/name: Red Hat .NET imagestreams.
+  charts.openshift.io/provider: Red Hat
```

Comment:
Other places use `openshift.io/provider-display-name: Red Hat, Inc.`. This seems a bit strange to me, but looking at other charts, `Red Hat` seems consistent here.

Reply:
Currently, these are the same labels as the 0.0.1 version.

@komish are there some labels we should be adding/removing/updating?

Reply:
Full list of provider-included labels is found here. https://github.com/redhat-certification/chart-verifier/blob/main/docs/helm-chart-annotations.md#provider-annotations

I think you're only missing the support URL one here, if you want to include it.

Reply:
I've added the support url and set it to https://github.com/redhat-developer/s2i-dotnetcore.